frontend/src/scenes/notebooks/Notebook/Notebook.tsx

```diff
@@ -25,7 +26,7 @@ export type NotebookProps = {
 
 const PLACEHOLDER_TITLES = ['Release notes', 'Product roadmap', 'Meeting notes', 'Bug analysis']
 
-export function Notebook({ shortId, editable = false, initialAutofocus = null }: NotebookProps): JSX.Element {
+export function Notebook({ shortId, editable = false, initialAutofocus = 'start' }: NotebookProps): JSX.Element {
```

Comment:
Is there a reason to change this value? I don't remember the specific example but I think I tried setting a default before and it caused unexpected behaviour in some places where the popover would open and you couldn't just start typing to append to the end of a notebook